webpack.config.js

```diff
@@ -203,8 +210,11 @@ const config = [
 
 // Support project overrides
 if (fs.existsSync('./project/webpack-project.config.js')) {
-  const projConfig = require('./project/webpack-project.config.js');
-  config[0].entry = Object.assign(config[0].entry, projConfig);
+    const projConfig = require('./project/webpack-project.config.js');
+
+    for (const [module, files] of Object.entries(projConfig)) {
```

Comment:
Do you have an example `webpack-project.config.js` so I can try and figure out if this part makes sense?

Reply:
its in the description
thats litterally the entire file

Reply:
okay, looks good to me.